bindings/kotlin/build.gradle.kts

```diff
@@ -16,11 +16,28 @@ dependencies {
 
 kotlin { jvmToolchain(21) }
 
-application { mainClass.set("ExampleKt") }
+// Generic task to run any example
+tasks.register<JavaExec>("example") {
+    classpath = sourceSets["main"].runtimeClasspath
+    jvmArgs = listOf("-Djna.library.path=${projectDir}/lib")
+    
+    // Get the example name from the command line argument -Pexample=<name>
+    val exampleProperty = "example"
+    inputs.property(exampleProperty, project.findProperty(exampleProperty) ?: "example")
+    
+    mainClass.set(provider {
+        val example = project.findProperty(exampleProperty)?.toString() ?: "example"
+        // Convert snake_case to CamelCase and append Kt
+        val className = example.split('_')
+            .map { it.replaceFirstChar { c -> c.uppercaseChar() } }
+            .joinToString("")
```

Comment:
Not needed anymore right?

Reply:
It is, the argument still needs to be the same for the Makefile `make bindings-example chain_id` and then we transform it to CamelCase to find the file